🐛(section) remove useless dependency to nesteditem migration (#991)
openfun/richie: CHANGELOG.md

## [Unreleased]

### Fixed

- Fix section migration "0005_migrate_sectionlist_to_nesteditem" that was

>   depending from NestedItem initial migration.

Reviewer: ```suggestion
  depending on NestedItem initial migration.
```

> - Fix section migration "0005_migrate_sectionlist_to_nesteditem" that was
>   depending from NestedItem initial migration.

Reviewer: ```suggestion
- Fix section migration 0005 that was depending on NestedItem initial migration.
```
